Dockerfile

```diff
@@ -1,11 +1,13 @@
 FROM node:12
 WORKDIR /usr/src/app
 COPY package.json .
-RUN npm install
-COPY index.ts .
+COPY package-lock.json .
+COPY tsconfig.json .
 COPY src src
+RUN npm ci
```

Comment:
`npm ci` should be used for reliable and fast installs - it deletes node_modules before starting and uses `package-lock.json`